mesheryctl/cmd/update.go

```diff
@@ -30,6 +37,74 @@ var updateCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
@anandrkskd let's update the short and long descriptions to include the fact the `mesheryctl` will be updated, too. 

See the text on this page for inspiration - https://meshery.layer5.io/docs/guides/mesheryctl#meshery-lifecycle-management